postman/SempoBlockchain_Dev.postman_environment.json

```diff
@@ -0,0 +1,19 @@
+{
+  "id": "6b629734-06b6-43a0-95f2-73543fe3732f",
+  "name": "SempoBlockchain_Dev",
+  "values": [
+    {
+      "key": "URL",
+      "value": "https://dev.withsempo.com/api/v1",
```

Comment:
the postman endpoints already have `/api/v1` in them, so this should just be the host eg `https://dev.withsempo.com/`. Actually, would probably change the name of this key to `HOST` rather than `URL` as it's more descriptive

Reply:
very good point! fixed and updated